docs/components/plugins/api/APISectionUtils.tsx

```diff
@@ -380,27 +393,72 @@ export const getTagData = (tagName: string, comment?: CommentData) =>
 export const getAllTagData = (tagName: string, comment?: CommentData) =>
   comment?.tags?.filter(tag => tag.tag === tagName);
 
+const getCleanPlatformName = (platform: CommentTagData) => {
+  if (platform.text.includes('ios')) return 'ios';
+  if (platform.text.includes('android')) return 'android';
+  if (platform.text.includes('web')) return 'web';
+  if (platform.text.includes('expo')) return 'expo';
+  return undefined;
+};
+
+const renderPlatformIcon = (platform: CommentTagData) => {
```

Comment:
Instead of making a function, we should make a component. General rule of thumb is that when making a function with the word `render` in it, it's likely a component.

```tsx
export function PlatformIcon(props: Props) {
  const { platform } = props;
  
  switch( ... ) {
    case '...':
      return <AppleIcon ... />
     // ...
  }
}
```

Also, I'd love to see a TypeScript `enum` for the platform names:

```ts
enum Platform {
  IOS = 'ios',
  ANDROID = 'android',
  // ...
}
```

Then we can name `getCleanPlatformName()` to `getPlatfromType()`, which would return one of the `Platform` enums.